.travis.yml

```diff
@@ -72,11 +72,8 @@ script:
 # Deploy debian packages to bintray
 deploy:
     provider: script
-    script: beaver bintray upload -N pkg/deb/*.deb
+    script: beaver bintray upload -N deb/*.deb
     skip_cleanup: true
     on:
         tags: true
-        condition: >
-            "$TRAVIS_OS_NAME" = "linux" -a
-            "$CC" = "gcc"
-
+        condition: $TRAVIS_OS_NAME = 'linux' && $CC = 'gcc'
```

Comment:
And do you know this is really a problem? Because it should work. I mean, putting it in one line seems good if it fits but the `-a` -> `&&` shouldn't be necessary if the condition is supposed to be pasted in at `if [[ ... ]]`.